### ISSUE TITLE
problem with installation

### ISSUE THREAD

**Issue by panagiotispir** [timduly4/pyglow#56]
hello I am trying to install pyglow and i fave the following issue.
When I execute make -C src/pyglow/models source I take the following response

```

rm -rf *~
make -Cdl_models/hwm93 clean
make[1]: Entering directory '/home/panos/pyglow/src/pyglow/models/dl_models/hwm93'
make[1]: *** No rule to make target 'clean'.  Stop.
make[1]: Leaving directory '/home/panos/pyglow/src/pyglow/models/dl_models/hwm93'
Makefile:47: recipe for target 'clean' failed
make: *** [clean] Error 2
```
plz help me I really want pyglow

**Comment by timduly4** [comment]
@panagiotispir This was a bug in the installation script, and is fixed here: https://github.com/timduly4/pyglow/commit/3f6137580b59a0e79aae8daad442ac2eee65b004

**Comment by timduly4** [comment]
@panagiotispir There was still an issue with the installation with respect to downloading static files-- I've fixed this in https://github.com/timduly4/pyglow/pull/57

**Comment by panagiotispir** [comment]
Hi again @timduly4 I clone you latest commit. 
First thank for yout attention.
Now I have another problem.
when I run the command `make -C src/pyglow/models source`
It starts to run and downloads models properly (I think) but after it stops with this:

`.
.
.
Downloading: http://irimodel.org/indices/apf107.dat to ./dl_models/iri16/apf107.dat
Downloading: http://irimodel.org/indices/ig_rz.dat to ./dl_models/iri16/ig_rz.dat
cp ./f2py/igrf11/* ./dl_models/igrf11/
cd ./dl_models/igrf11/; make source
make[1]: Entering directory '/home/panoslptp/pyglow/src/pyglow/models/dl_models/igrf11'
make patch_igrf11;	
make[2]: Entering directory '/home/panoslptp/pyglow/src/pyglow/models/dl_models/igrf11'
patch igrf11.f -i igrf11.patch -o igrf11_modified.f
patching file igrf11_modified.f (read from igrf11.f)
make[2]: Leaving directory '/home/panoslptp/pyglow/src/pyglow/models/dl_models/igrf11'
make sig;
make[2]: Entering directory '/home/panoslptp/pyglow/src/pyglow/models/dl_models/igrf11'
f2py -m igrf11py -h sig_file.pyf igrf11_modified.f only: igrf11syn : | tee out1
/bin/sh: 1: f2py: not found
make[2]: Leaving directory '/home/panoslptp/pyglow/src/pyglow/models/dl_models/igrf11'
make patch_sig;
make[2]: Entering directory '/home/panoslptp/pyglow/src/pyglow/models/dl_models/igrf11'
patch sig_file.pyf -i sig.patch -o sig_file_patched.pyf
patching file sig_file_patched.pyf (read from sig_file.pyf)
Hunk #1 FAILED at 13.
1 out of 1 hunk FAILED -- saving rejects to file sig_file_patched.pyf.rej
Makefile:23: recipe for target 'patch_sig' failed
make[2]: *** [patch_sig] Error 1
make[2]: Leaving directory '/home/panoslptp/pyglow/src/pyglow/models/dl_models/igrf11'
Makefile:29: recipe for target 'source' failed
make[1]: *** [source] Error 2
make[1]: Leaving directory '/home/panoslptp/pyglow/src/pyglow/models/dl_models/igrf11'
Makefile:23: recipe for target 'source' failed
make: *** [source] Error 2
make: Leaving directory '/home/panoslptp/pyglow/src/pyglow/models'
`
maybe something in the Makefile? but I can't locate which is the problem.
thanks in advance

**Comment by timduly4** [comment]
@panagiotispir , it looks like you don't have `f2py` installed, which usually is integrated with `numpy`.

Let's make sure you have a `numpy` version that is up to date:
```
~ $ python -c "import numpy; print(numpy.__version__)"
1.13.1
~ $ which f2py
/usr/local/bin/f2py
```

Try installing `numpy` if the above doesn't work.

**Comment by panagiotispir** [comment]
I run `pip install numpy` instead of `pip install python-numpy` and it seems like "numpy" didn't have f2py as I thought.
but with python-numpy it's ok.
Thank you very much.
I am newbie on linux and python, so you save a lot of time for me!!!!

**Comment by timduly4** [comment]
@panagiotispir that's great to hear.  Happy to help! 🎉

**Comment by timduly4** [comment]
@panagiotispir I also added your instructions for installing `f2py` on the front page: https://github.com/timduly4/pyglow/pull/58

**Comment by panagiotispir** [comment]
Hello @timduly4 I was planning to sent you again about my previous message but you were faster.
Yesterday I installed pyglow on another PC and `pip install numpy` worked fine and f2py installed properly instead of `pip install python-numpy` which seems to be unrecognized.
So, I concluded that the best way is to run `which f2py` to realize if f2py is installed. In case of f2py is missing `pip install numpy` or `pip install python-numpy` should make the job done.
